charts/sn-platform/templates/prometheus/prometheus-configmap.yaml

```diff
@@ -155,6 +155,11 @@ data:
 {{- with .Values.prometheus.extraScrapeConfigs }}
 {{ toYaml . | indent 4 }}
 {{- end }}
+{{- end }}
+{{- if .Values.prometheus.extraRemoteWrite -}}
```

Comment:
The configuration item key `remote_write` should be added, and it is at the same level as the key`global`

Reply:
Yes, this is generated config file

```
  prometheus.yml: |
    global:
      scrape_interval: 15s
    rule_files:
      - 'rules.yml'
    alerting:
      alertmanagers:
      - static_configs:
        - targets: ['pulsar19-sn-platform-alert-manager:9093']
        path_prefix: /
    scrape_configs:
    - job_name: 'prometheus'
      static_configs:
      - targets:
        - '127.0.0.1:9090'
      metrics_path: /metrics
    - job_name: 'kubernetes-pods'
      bearer_token_file: /pulsar/tokens/client/token
      kubernetes_sd_configs:
      - role: pod
      relabel_configs:
      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
        action: keep
        regex: true
      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_path]
        action: replace
        target_label: __metrics_path__
        regex: (.+)
      - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
        action: replace
        regex: ([^:]+)(?::\d+)?;(\d+)
        replacement: $1:$2
        target_label: __address__
      - action: labelmap
        regex: __meta_kubernetes_pod_label_(.+)
      - source_labels: [__meta_kubernetes_namespace]
        action: replace
        target_label: kubernetes_namespace
      - source_labels: [__meta_kubernetes_pod_label_component]
        action: replace
        target_label: job
      - source_labels: [__meta_kubernetes_pod_name]
        action: replace
        target_label: kubernetes_pod_name
      metric_relabel_configs:
    - job_name: 'kubernetes-nodes'
      scheme: https
      kubernetes_sd_configs:
        - role: node

      tls_config:
        ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
      bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token

      relabel_configs:
        - action: labelmap
          regex: __meta_kubernetes_node_label_(.+)
        - target_label: __address__
          replacement: kubernetes.default.svc:443
        - source_labels: [__meta_kubernetes_node_name]
          regex: (.+)
          target_label: __metrics_path__
          replacement: /api/v1/nodes/${1}/proxy/metrics
    - job_name: 'kubernetes-cadvisor'
      scheme: https
      kubernetes_sd_configs:
        - role: node

      tls_config:
        ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
      bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token

      relabel_configs:
        - action: labelmap
          regex: __meta_kubernetes_node_label_(.+)
        - target_label: __address__
          replacement: kubernetes.default.svc:443
        - source_labels: [__meta_kubernetes_node_name]
          regex: (.+)
          target_label: __metrics_path__
          replacement: /api/v1/nodes/${1}/proxy/metrics/cadvisor
    remote_write:
    - basic_auth:
        password: secret-password
        username: admin
      url: http://prometheus-service.prometheus.svc.cluster.local:9090/api/v1/write
```